ui/lib/pki/addon/routes/issuers/generate-root.js

```diff
@@ -1,3 +1,21 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class PkiIssuersGenerateRootRoute extends Route {}
+export default class PkiIssuersGenerateRootRoute extends Route {
+  @service secretMountPath;
+  @service store;
+
+  model() {
+    return this.store.createRecord('pki/action', {});
```

Comment:
do we always have to pass an empty object here? I've noticed it sometimes and wondered if I'm missing something 😅

Reply:
Nope, looks like that's optional